src/sentry/incidents/models/alert_rule.py

```diff
@@ -585,14 +601,37 @@ class Meta:
 
 
 @register_alert_subscription_callback(AlertRuleMonitorType.ACTIVATED)
-def clean_expired_alerts(subscription: QuerySubscription) -> bool:
+def update_alert_activations(
+    subscription: QuerySubscription, alert_rule: AlertRule, value: float
+) -> bool:
+    activations = alert_rule.activations.filter(
+        finished_at=None, query_subscription_id=subscription.id
+    )
+    for activation in activations:
+        activation.update(metric_value=value)
+
+    return True
```

Comment:
why does this return a boolean?

Reply:
I think this is just a success/failure indicator?
was following the existing registered callback 😅 
but i think you're right. doesn't need any return response i suppose

Reply:
yeah, the idea was that later on we could iterate through all the responses to the registry and say if they were successfully executed or not.